### PR TITLE
Upgrade to Node 20

### PR DESCRIPTION
**Pull request by lb-**
Node 20 is now the active LTS, while Node 18 will have security updates until April 2025 it would be good to be on the latest LTS if possible.

Once merged, I'll put PRs up for our other projects (e.g. Docker Wagtail develop).

### CODE REVIEW

**Comment by lb-** [comment]
Not sure if there are specific permissions needed for changing the CI?

```
Build-agent version 1.0.207522-d583bc78 (2023-10-23T14:33:36+0000).
System information:
 Server Version: 20.10.18
 Storage Driver: overlay2
  Backing Filesystem: xfs
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Kernel Version: 5.15.0-1039-aws
 Operating System: Ubuntu 20.04.6 LTS
 OSType: linux
 Architecture: x86_64

Starting container cimg/node:20.12
Warning: No authentication provided, using CircleCI credentials for pulls from Docker Hub.
  image cache not found on this host, downloading cimg/node:20.12

Error response from daemon: manifest for cimg/node:20.12 not found: manifest unknown: manifest unknown
```

**Comment by zerolab** [comment]
@lb- looking at https://circleci.com/developer/images/image/cimg/node, it looks like 20.9.0 is the latest they have in the 20.x series

**Comment by lb-** [comment]
Ahh thanks. I completely missed that I had the wrong specific version.

File: .circleci/config.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
 
   frontend:
     docker:
-      - image: cimg/node:18.12
+      - image: cimg/node:20.12
```
Comment (lb-) [inline .circleci/config.yml:36]:
```suggestion
      - image: cimg/node:20.9
```

**Comment by lb-** [comment]
OK. Good for a review now.

**Review by allcaps — APPROVED** [review]

**Comment by lb-** [comment]
Thanks @allcaps  I will add a changelong entry and merge.

**Pull request merged**